sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeBatchActionsOperationDetail.java

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.ai.textanalytics.implementation.AnalyzeBatchActionsOperationDetailPropertiesHelper;
+
+import java.time.OffsetDateTime;
+
+/**
+ * The {@link AnalyzeBatchActionsOperationDetail} model.
+ */
+public final class AnalyzeBatchActionsOperationDetail {
```

Comment:
Missing @Immutable?

Reply:
We have ***PropertyHelper to set the properties internally. So the Immutable annotation can't be used.